webapp/security/views.py

```diff
@@ -31,14 +33,19 @@
 security_api = SecurityAPI(session=session)
 
 
-def get_processed_details(notice):
-    pattern = re.compile(
-        r"(?<![a-zA-Z0-9-_/])((cve|CVE-)\d{4}-\d{4,7})(?!\.html)", re.MULTILINE
+def get_attention_banner(details):
```

Comment:
I think we should move this into helpers.py as well, it's the only function left in this file that doesn't render a view I believe